astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -30,12 +29,9 @@
 from astropy.io.ascii.ecsv import DELIMITERS
```

Comment:
lint: because of the change below, `AstropyWarning` is no longer used in this module, so the import can be removed (`AstropyUserWarning`, however, is still used elsewhere in the file)

Reply:
Done!